bin/boilerplate.rs

```diff
@@ -93,6 +99,44 @@ impl Harness {
                             if !app.on_key(input) {
                                 running = false;
                             }
+                            info!("alt_pressed: {:?}", input);
+                            match input.virtual_keycode {
+                                Some(Key::LControl) => {
+                                    alt_pressed = input.state == ElementState::Pressed;
+                                }
+                                _ => {}
+                            }
+
+                        }
+                        glutin::WindowEvent::MouseWheel {delta, ..} => {
+                            app.on_mouse_wheel(delta)
+                        }
+                        glutin::WindowEvent::CursorMoved {position, ..} => {
+                            if mouse_pressed {
```

Comment:
similarly, let's just pass it down to the application directly.
I imagine a use case we might want to get that would highlight parts of the level based on where the mouse is, even when no buttons are pressed